src/LiipImagine/FilterResolverRegistry.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\LiipImagine;
+
+use Liip\ImagineBundle\Imagine\Cache\Resolver\ResolverInterface;
+use Liip\ImagineBundle\Imagine\Filter\FilterConfiguration;
+
+class FilterResolverRegistry implements ResolverRegistryInterface
+{
+    /**
+     * @var ResolverInterface[]
+     */
+    private $resolvers;
+    /**
+     * @var FilterConfiguration
+     */
+    private $filterConfiguration;
+    /**
+     * @var string
+     */
+    private $defaultResolver;
+
+    /**
+     * @param FilterConfiguration $filterConfiguration
```

Comment:
Please remove line 33

Reply:
@greg0ire then the whole doc-block must be removed or its inconsistent with the method signature, you prefer that?

Reply:
it is, and IDE's will not render the doc-bloc properly, imo adding to the often already confusing (or none existing documentation)

![image](https://user-images.githubusercontent.com/1032205/45077724-36080380-b0ee-11e8-827e-478ca37996e3.png)

as shown in the screenshot it's not declaring the parameter, I know there is a minimal documentation policy (for whatever reason), but in the 1 week i've been busy with sonata that has already frustrated me (and surly other developers) more than anything else, maybe because i'm using PHPStorm and it's highlighting every inconsistency there is, but PHPStorm is as far as i know the most widely used IDE.

I'll remove it, but that's just my 2ct.

Reply:
I'm at my desk now, can answer in a more lengthy fashion. Ce phpdoc contains the parameter name, so there will not be any confusion between parameters if that's what you are afraid of.